### ISSUE TITLE
(1.4.0) Crashes on start

### ISSUE THREAD

**Issue by LetItGlow** [logue/NgsPacker#8]
Just starting the program gives me an error message "pso2.exe not found" and crashes a couple seconds afterwards.

WinDBG gives me this.
`STACK_COMMAND:  ** Pseudo Context ** ManagedPseudo ** Value: ffffffff ** ; kb
SYMBOL_NAME:  PresentationFramework!System.Windows.Window.ShowDialog+39f
MODULE_NAME: PresentationFramework
IMAGE_NAME:  PresentationFramework.dll
FAILURE_BUCKET_ID:  CLR_EXCEPTION_80070057_PresentationFramework.dll!System.Windows.Window.ShowDialog
OS_VERSION:  10.0.19041.1
BUILDLAB_STR:  vb_release
OSPLATFORM_TYPE:  x64
OSNAME:  Windows 10
IMAGE_VERSION:  6.0.522.21509
FAILURE_ID_HASH:  {2d609b9f-4ad2-939d-3210-9a8f2c55089f}
`

It crashes "onShow"... well that's interesting.

**Comment by logue** [comment]
Is Window 11? In the case of 10, it may not work unless it is updated to the latest version.

**Comment by LetItGlow** [comment]
I am using Windows 10 22H2. Can't get more up-to-date.

**Comment by logue** [comment]
I have migrated to dotnet7 from 1.4, but is that not enough?

⚠ The CSV output function is broken because PSO2NGS now includes unencrypted raw data.

**Comment by Leayal** [comment]
@LetItGlow Maybe I was late. I've created a fork and resolved this issue. [The pre-built binaries can be found here](https://github.com/Leayal/NgsIceArchive/releases/tag/1.5.0-alpha).

I also posted [this pull request](https://github.com/logue/NgsPacker/pull/9) which resolved the issue. But it seems like the original author is busy.

tl;dr: It wasn't the .NET version's fault, it's rather setting the unrecognized parameters like Window Border Radius and such.
These parameters are only available in Win11, Win10 will not reconize and the WinAPI will return error. However, the error/exception isn't handled in .NET codebase so the .NET app just simply crashes due to unhandled exception.
